--- flatpak-builder-1.4.3/configure.ac.orig	2024-04-18 20:54:09.385267515 +0200
+++ flatpak-builder-1.4.3/configure.ac	2024-04-18 21:07:28.140940284 +0200
@@ -38,7 +38,7 @@ if test "x$FLATPAK" = xfalse; then
 else
    FLATPAK_VERSION=`$FLATPAK --version | sed 's,.*\ \([0-9]*\.[0-9]*\.[0-9]*\)$,\1,'`
    AX_COMPARE_VERSION([$FLATPAK_REQS],[gt],[$FLATPAK_VERSION],
-                      [AC_MSG_ERROR([You need at least version $FLATPAK_REQS of flatpak, your version is $FLATPAK_VERSION])])
+                      [AC_MSG_WARN([You need at least version $FLATPAK_REQS of flatpak, your version is $FLATPAK_VERSION])])
 fi
 
 AC_CHECK_PROG([APPSTREAMCLI], [appstreamcli], [appstreamcli], [false])
@@ -46,7 +46,7 @@ if test "x$APPSTREAMCLI" = xfalse; then
    AC_MSG_ERROR([You need appstreamcli installed])
    APPSTREAMCLI_VERSION=`$APPSTREAMCLI --version | sed 's,.*\ \([0-9]*\.[0-9]*\.[0-9]*\)$,\1,'`
    AX_COMPARE_VERSION([$APPSTREAMCLI_REQS],[gt],[$APPSTREAMCLI_VERSION],
-                      [AC_MSG_ERROR([You need at least version $APPSTREAMCLI_REQS of appstreamcli, your version is $APPSTREAMCLI_VERSION])])
+                      [AC_MSG_WARN([You need at least version $APPSTREAMCLI_REQS of appstreamcli, your version is $APPSTREAMCLI_VERSION])])
 fi
 
 AC_MSG_CHECKING([whether appstreamcli has compose support])
@@ -54,7 +54,7 @@ AS_IF([appstreamcli compose --help >/dev
       [AC_MSG_RESULT(yes)],
       [
        AC_MSG_RESULT(no)
-       AC_MSG_ERROR([appstreamcli must have compose support enabled and installed])
+       AC_MSG_WARN([appstreamcli must have compose support enabled and installed])
       ])
 
 LT_PREREQ([2.2.6])
